src/Parser.js

```diff
@@ -244,8 +244,8 @@ export default class Parser {
                 denomNode = new ParseNode("ordgroup", denomBody, this.mode);
             }
 
-            const value = this.callFunction(funcName, [numerNode, denomNode], []);
-            return [new ParseNode(value.type, value, this.mode)];
```

Comment:
It's a pity: we never passed in the source location when constructing these nodes, and now we're spreading that behavior everywhere.